hnspolicy.go

```diff
@@ -9,18 +9,18 @@ type PolicyType = hns.PolicyType
 
 // RequestType const
 const (
-	Nat                  = hns.Nat
-	ACL                  = hns.ACL
-	PA                   = hns.PA
+	NAT                  = hns.NAT
+	QOS                  = hns.QOS
 	VLAN                 = hns.VLAN
 	VSID                 = hns.VSID
-	VNet                 = hns.VNet
-	L2Driver             = hns.L2Driver
-	Isolation            = hns.Isolation
-	QOS                  = hns.QOS
+	ACL                  = hns.ACL
+	PA                   = hns.PA
 	OutboundNat          = hns.OutboundNat
+	ROUTE                = hns.ROUTE
+	L2Driver             = hns.L2Driver
+	SWITCHEXTENSION      = hns.SWITCHEXTENSION
```

Comment:
I really don't think there should be capitalisation of things that aren't abbreviations. `SWITCHEXTENSION` and `ROUTE` just look really odd.